modules/ovn-kubernetes-logical-architecture-con.adoc

```diff
@@ -0,0 +1,68 @@
+:_content-type: CONCEPT
+[id="ovn-kubernetes-logical-architecture-con_{context}"]
+= OVN-Kubernetes logical architecture
+
+OVN is a network virtualization solution. It creates logical switches and routers. These switches and routers are interconnected to create any network topologies. When you run `ovnkube-trace` with the log level set to 2 or 5 the OVN-Kubernetes logical components are exposed. The following diagram shows how the routers and switches are connected in {product-title}.
+
+.OVN-Kubernetes router and switch components
+image::299_OpenShift_OVN-Kubernetes_arch_0123_2.png[OVN-Kubernetes logical architecture]
+
+The key components involved in packet processing are:
+
+Gateway routers:: Gateway routers sometimes called L3 gateway routers, are typically used between the distributed routers and the physical network. Gateway routers including their logical patch ports are bound to a physical location (not distributed), or chassis. The patch ports on this router are known as l3gateway ports in the ovn-southbound database (`ovn-sbdb`). For example in the following `ovnkube-trace` snippet the gateway routers are prefixed with `GR_`:
++
+[source,terminal]
+----
```

Comment:
I think it's more practical to give a command that customers can more easily repeat for this like:
[root@openshift-jumpserver-0 debug-scripts]# ./network-tools ovn-db-run-command ovn-nbctl lr-list
Leader pod is ovnkube-master-nbrmw
74ccc2cc-6e38-49a7-bcad-c40eea2345f0 (GR_openshift-master-0)
77ce4ade-ddbc-4fc3-820b-f6b1424cbd4a (GR_openshift-master-1)
846bdb13-55d3-4165-bfb0-b7d290477c9e (GR_openshift-master-2)
d6a6bfc6-6ab4-4b55-9de7-abe03087b09b (GR_openshift-worker-0)
f62c6d6f-8818-49ca-be4c-b8df68987855 (GR_openshift-worker-1)
2842e138-66cb-404f-9031-c6afab146be6 (GR_openshift-worker-2)
51394e7e-f57e-4271-ae5b-896848ac5160 (ovn_cluster_router).   

Of course this does show both types of routers.

Reply:
I'm using a tool from https://github.com/openshift/network-tools to make finding the master pod easier.  the important part of that command is 'ovn-nbctl lr-list'

Reply:
@CFields651 I do tend to agree with you, but I believe we discourage the installation of third party tool in our documentation. I know they are support concerns around using jq so I expect there may be issues with network-tools though maybe not. I will check with OCP doc team